samples/api/hello_triangle/hello_triangle.cpp

```diff
@@ -685,7 +685,7 @@ void HelloTriangle::init_render_pass()
  * @param path The path for the shader (relative to the assets directory).
  * @returns A VkShaderModule handle. Aborts execution if shader creation fails.
  */
-VkShaderModule HelloTriangle::load_shader_module(const char *path)
+VkShaderModule HelloTriangle::load_shader_module(const std::string path)
```

Comment:
might add a & in there; this copies the path every call needlessly.

Reply:
Good catch. Sometimes MSVC hints you at that, and sometimes it doesn't ^^